src/components/Image/Image.mdx

```diff
@@ -109,6 +111,40 @@ the `onError` function callback.
 
 <Canvas of={ImageStories.HTMLAttributes} />
 
+## Fallback Image
+
+Sometimes images don't load due to network issues or the image URL is incorrect.
+Set a fallback image source through the `fallbackSrc` prop. Internally, the
+`Image` component will call the `img` element's `onerror` attribute with a
+callback function to load the `fallbackSrc` image. Additionally, a warning will
+be logged to the console when the error occurs and the fallback image is loaded.
+
+If an _additional_ action needs to be performed when the fallback image is
+loaded, pass a callback function to the `onError` prop.
+
+**IMPORTANT**: If a custom image is passed through the `component` prop, then
+`fallbackSrc` and `onError` won't work. It is expected that the custom image
```

Comment:
this is a little confusing to me. `onError` won't work but it's expected?

Reply:
Hmm yes. I can add another example. The following is what I mean but does it make sense?

```
**IMPORTANT**: If a custom image, such as the Next.js `Image`, is passed through the `component`
prop, then the `fallbackSrc` and `onError` props won't work. It is expected that the custom image
that is passed in the implements the `onError` attribute.

<Source
  code={`
import { Image } from "@nypl/design-system-react-components";
import Image as NextImage from "next/image";

// ...
    <Image
      component={{
         <NextImage
            src="https://placekitten.com/540/420"
            onError={(event) => {
              // implement your own fallback event
              event.target.src =  "https://loremflickr.com/540/420/cat"
            }}
         />
      }}
    />
`}
  language="jsx"
/>
```